readme.md

```diff
@@ -22,7 +22,8 @@ composer require tightenco/collect
 ## FAQ
  - **Will this develop independently from Illuminate's Collections?**  
     No. Right now it's split manually, but the goal is for it shortly to be split automatically to keep it in sync with Laravel's Collections, even mirroring the release numbers.
- - **Why isn't this just under the Illuminate namespace?**  
-    Because that would require adding a lot of complexity and duplication to Illuminate\Support's internal structure. One day Taylor may choose to do that, but right now he hasn't and doesn't have any immediate plans to do so. If he does, we'll deprecate this package and point to the core version.
+ - **Why isn't the package named illuminate/collect?**  
+    It's not an official Laravel package so we don't want to use the namespace reserved by Laravel packages.  
+    One day `Collection` may be extracted from `illuminate/support` to a new package. If so, we'll deprecate this package and point to the core version.
```

Comment:
One day `Collection` may be extracted from `Illuminate\Support` to a new package. If so, we'll deprecate this package and point to the core version.